roles/satellite-clone/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # more info about these variables can be found in satellite-clone-vars.sample.yml in the root directory of this project
-satellite_version: 6.2
+satellite_version: 6.1
```

Comment:
Having a default satellite_version is not necessary.  You can set 6.1 or 6.2 during your answers file check